release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -3422,4 +3422,28 @@ $ oc adm release info 4.11.34 --pullspecs
 [id="ocp-4-11-34-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
```

Comment:
```suggestion
For instructions to update an {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```

Reply:
This is from a template so I'd rather not deviate from it. Though I like your suggestion.